### DISCUSSION TITLE
Hacktober Fest 2024

### DISCUSSION THREAD

**Discussion by Nj221102** [ImageMagick/ImageMagick#7665] [General] [answered]
Hi everyone,  
I’m Nitish Jha, a computer science student currently participating in Hacktoberfest. I’m interested in contributing to ImageMagick as I already have experience working on C projects. However, I have a few questions.

Typically, Hacktoberfest issues are tagged with something like `hacktoberfest-24` or a similar label, but I haven’t noticed any such tags here. Could you please confirm if this repository is participating in Hacktoberfest? If so, since there are no specific tags, does that mean all issues are eligible for contribution under Hacktoberfest?

**Comment by dlemstra** [comment, marked as answer]
Our repository is tagged for Hacktober Fest so yes any issue/pre can qualify for that.

**Reply by Nj221102** [reply]
thanks for info.
